app/assets/javascripts/discourse/app/widgets/toggle-topic-summary.js

```diff
@@ -100,7 +100,7 @@ export default createWidget("toggle-topic-summary", {
     const icon = canRegenerate ? "sync" : "magic";
 
     return this.attach("button", {
-      className: "btn btn-primary topic-strategy-summarization",
```

Comment:
Can we add the class if `attrs.hasTopRepliesSummary` is true? We want to differentiate between both buttons.